oci-unit-tests/cassandra_test.sh

```diff
@@ -0,0 +1,140 @@
+# shellcheck shell=dash
+
+# shellcheck disable=SC1090
+. "$(dirname "$0")/helper/test_helper.sh"
+. "$(dirname "$0")/helper/common_vars.sh"
+
+# cheat sheet:
+#  assertTrue $?
+#  assertEquals ["explanation"] 1 2
+#  oneTimeSetUp()
+#  oneTimeTearDown()
+#  setUp() - run before each test
+#  tearDown() - run after each test
+
+CQLSH_DOCKER_IMAGE="cassandra-cqlsh:test"
+
+oneTimeSetUp() {
+    # Remove image before test.
+    remove_current_image
+
+    # Make sure we're using the latest OCI image.
+    docker pull --quiet "${DOCKER_IMAGE}" > /dev/null
+
+    docker network create "$DOCKER_NETWORK" > /dev/null 2>&1
+
+    # build image with cqlsh client
+    docker build -t $CQLSH_DOCKER_IMAGE ./cassandra_test_data > /dev/null 2>&1
+}
+
+setUp() {
+    id=$$
+}
+
+oneTimeTearDown() {
+    docker network rm "$DOCKER_NETWORK" > /dev/null 2>&1
+    # remove local image with cqlsh client
+    docker rmi -f $CQLSH_DOCKER_IMAGE > /dev/null 2>&1
+}
+
+tearDown() {
+    if [ -n "${container}" ]; then
+        stop_container_sync "${container}"
+    fi
+    if [ -n "${volume}" ]; then
+        docker volume rm "${volume}" > /dev/null 2>&1
+    fi
+}
+
+# Helper function to execute cassandra with some common arguments.
+docker_run_server() {
+    docker run \
+     --network "$DOCKER_NETWORK" \
+     --rm \
+     -d \
+     --name cassandra_test_${id} \
+     "$@" \
+     "${DOCKER_IMAGE}"
+}
+
+# Helper function to execute cqlsh with some common arguments.
+docker_run_client() {
+    # mount data into the container
+    cqlsh_file="${PWD}/cassandra_test_data/hello-cassandra.cqlsh"
```

Comment:
We need an absolute path to mount the volume here (this is also how this is being done for the nginx tests). Any reason for refraining from using $PWD here?

Reply:
You will get an absolute path when using `"$(dirname "$0")"`.

The reason for not using `$PWD` is because we want to be able to execute these scripts from other directories.  If you use `$PWD`, its value will change depending on where you are executing from.  If you use the `dirname` trick, it will always yield the same path, which is where the test script is located.

Reply:
Ah, sorry, you will have to call `$(realpath $(dirname $0))` to get the absolute path.  But the rest of the comment still applies :-).

Reply:
Sounds good! Done :)